src/components/TodoList/TodoList.jsx

```diff
@@ -1,15 +1,15 @@
-import TodoListItem from "./TodoListItem";
-import "./App.css";
+import TodoListItem from "../TodoListItem/TodoListItem";
+import style from "./TodoList.module.css";
 
 function TodoList({ todoList, onRemoveTodo }) {
   return (
     <>
       <br />
-      <table className="todo-table">
+      <table className={style.table}>
         <thead>
           <tr>
-            <th>Title</th>
-            <th>Actions</th>
+            <th className="todo-table th">Title</th>
```

Comment:
The className values are written as a single string "todo-table th", but they don't seem to reference any actual styles. Also, the combination of multiple classes within quotes could be misleading if there isn't a style defined in your CSS that matches both todo-table and th. If these class names are not needed, you could simply remove them:

Reply:
Thanks, I made a change.